src/connection.js

```diff
@@ -32,12 +33,15 @@ const DEFAULT_CANCEL_TIMEOUT = 5 * 1000;
 const DEFAULT_PACKET_SIZE = 4 * 1024;
 const DEFAULT_TEXTSIZE = '2147483647';
 const DEFAULT_PORT = 1433;
+const DEFAULT_DATEFIRST = 7;
 const DEFAULT_TDS_VERSION = '7_4';
+const DEFAULT_LANGUAGE = 'us_english';
+const DEFAULT_DATEFORMAT = 'mdy';
 
 class Connection extends EventEmitter {
   constructor(config) {
     super();
-
+    var self = this;
```

Comment:
We could use this, since we dont use this in some loop and constructor is not called offten.